src/components/templates/Playa/Playa.tsx

```diff
@@ -40,7 +40,7 @@ import { WithId } from "utils/id";
 // import { updateLocationData } from "utils/userLocation";
```

Comment:
remove all of the commented code, since we are not going to uncomment it in the future. Please, do it to all of the occurances

Reply:
removed d0e862618e8e3afacd238ae21284bf4230067a79